resdx/defrost.py

```diff
@@ -20,7 +20,7 @@ def __init__(self,time_fraction = lambda conditions : fr_u(3.5,'min')/fr_u(60.0,
                     resistive_power = 0,
                     control = DefrostControl.TIMED,
                     strategy = DefrostStrategy.REVERSE_CYCLE,
-                    high_temperature=fr_u(41,"°F"),  # Maximum temperature for defrost operation
+                    high_temperature=fr_u(47,"°F"),  # Maximum temperature for defrost operation according Hugh I. Henderson et al: Savings Calculations for Residential Air Source Heat Pumps (NYSERDA and NYS Department of Public Service)
```

Comment:
I don't know if we want to sue this for our default. Let's stick with 41 for now and we can run it by the working group.

Reply:
Updated.